marketplace/bynder-dialog/src/BynderDialog.js

```diff
@@ -0,0 +1,54 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+
+const BYNDER_SDK_URL =
+  'https://d8ejoa1fys2rk.cloudfront.net/modules/compactview/includes/js/client-1.5.0.min.js';
+
+export default class Dialog extends React.Component {
+  static propTypes = {
+    sdk: PropTypes.object.isRequired
+  };
+
+  componentDidMount() {
+    this.loadBynderScript();
+    this.props.sdk.window.updateHeight();
+    document.addEventListener('BynderAddMedia', e => {
+      const results = Array.isArray(e.detail) ? e.detail : [];
+      const assets = results.map(asset => ({
+        id: asset.id,
+        src: asset.thumbnails['webimage']
+      }));
+      this.props.sdk.close(assets);
+    });
+  }
+
+  loadBynderScript = () => {
```

Comment:
This doesn't need to be an arrow function. (doesn't use `this`)

Also why do this `componentDidMount` instead of adding the JS file directly to the HTML?